webpack.config.js

```diff
@@ -10,11 +10,15 @@ const terser = require('terser');
 const fs = require('fs');
 const pkg = require('./package.json');
 const cp = require('child_process');
+const optional = require('optional');
 
 const isDev = (process.env.NODE_ENV === 'development');
 const isTest = (process.env.NODE_ENV === 'test');
 const isProd = (process.env.NODE_ENV === 'production');
 
+const linkedPackages = isDev ? optional('./config/linked-packages').packages || {} : {};
```

Comment:
Should this be:
```suggestion
const linkedPackages = (isDev || isTest) ? optional('./config/linked-packages').packages || {} : {};
```
to handle the case that a unit test's success depends on changes in a locally linked and yet unpublished package?

Reply:
Actually, I see that will probably have implications on the CI servers. Perhaps we could use a `isCI` check? Not sure if that's adding too much complexity though.

Reply:
Well, in my description, I noted that I'd initially excluded TEST on purpose.  I suppose I'm good either way though.  What I wanted to avoid, and what has happened, is having tests pass, then having them fail in CI since the dependancy wasn't published.

That's just my workflow, though -- I like to get the tests working in the linked package, then publish and run the test-watcher in `blip`.

I'll commit the suggestion.  Makes sense to have it allow for the workflow you've outlined

Reply:
Ah, you're right about the CI issue.  And I've already committed the suggestion while you were typing!  Let me look into the CI check option.  Otherwise, we could add a secondary means of running the tests with the linked packages, but not by default, which may be less flaky than whatever CI check I come up with.

Thoughts?

Reply:
Actually, I don't think it would be an issue, since the linked-packages.js file is excluded from VC, and would not be present in CI builds, and would thus just spread an empty object into the aliases as intended.

Gonna revert my revert :)

Reply:
Tests passing in CI (build is not finished as of time of writing this, but the `npm test` portion has) with latest commit.  I think we're good on this front.

Reply:
Cool, seems good to me. If forgetting to publish packages becomes a common issue, then we can look into ways to solve it (whether that's moving to local/linked-specific testing `npm scripts` or something else).

Reply:
It's honestly not a problem for me, and it would only result a failed CI build.  I have seen interns trip over it, so that's all I had in mind -- kind of avoiding that scenario for those who may not be mindful of it.

After giving it some thought, I see no real downside to it. and could be seen as a convenience at times.